### DISCUSSION TITLE
Is the live preview generated by Zettlr or Pandoc?

### DISCUSSION THREAD

**Discussion by pbsds** [Zettlr/Zettlr#4753] [Q&A] [answered]
If i use say a very weird custom pandoc filter, will the results show up in the live preview in Zettlr?

**Comment by nathanlesage** [comment, marked as answer]
If by the live preview you mean the pre-rendered elements directly in the editor, no: those are built from the editor's syntax tree that can differ from Pandoc; custom plugins won't show up unless you export. There are (long term!) ideas to implement a more proper preview directly via Pandoc at some undetermined later point, though.
